onaio/kujaku
# Add sample activity showing how to add a feature select status

Related to #183 but not fixing it

## How to test it

- Go to the **Feature Click Status** activity from the Navigation menu
- Navigate to an area that you know has buildings or any polygonal feature, click on it, the feature should turn orange
- Clicking on another feature should also turn it to orange. This just enables you to select multiple features
